docs/synapse/userguides/storm_ref_data_mod.rstorm

```diff
@@ -180,6 +181,31 @@ where the variable may contain unexpected values. For example:
 
 See the :ref:`type-syn-tag` section of the :ref:`storm-ref-type-specific` for additional detail on tags / ``syn:tag`` forms.
 
+Conditional Edit Operators
+++++++++++++++++++++++++++
+
+The conditional edit operators ( ``*unset=`` and ``*$<varvalue>=`` ) can be used to only set properties when certain
```

Comment:
```suggestion
The conditional edit operators ( ``*unset=`` and ``*$<varname>=`` ) can be used to only set properties when certain
```

Reply:
Yeah I was undecided between varname and varvalue since we do allow more than just a variable name here, but I may be overthinking it.